.ansible-lint

```diff
@@ -0,0 +1,35 @@
+exclude_paths:
```

Comment:
I don't know if this is significant, but I pulled up the docs, and .ansible-lint is supposed to be a titled yaml file: 
Probably minor detail: but here are the first 2 lines of the example:
---
# .ansible-lint

https://ansible.readthedocs.io/projects/lint/configuring/#specifying-configuration-files

Reply:
git ate my "---" on the first line.

Reply:
Its a good point, but the docs also say it does not have to be.
We have a working `ansible-lint` on our playbook repo [here](https://github.com/IBM/z_ansible_collections_samples/blob/main/.ansible-lint)
Same docs you link to state this: 
```
Specify Ansible-lint configuration in either .ansible-lint, .config/ansible-lint.yml, or .config/ansible-lint.yaml in your current working directory.
```
and I am glad to see the docs up, many read-the-docs are down due to a migration configuration, some are addressing it sooner than others, docs were down for me when i was doing this work. Consider this resolved then.